lib/pages/calendar_attendance_page.dart

```diff
@@ -0,0 +1,180 @@
+import 'package:flutter/material.dart';
+import 'package:table_calendar/table_calendar.dart';
+
+class CalendarAttendancePage extends StatefulWidget {
+  @override
+  _CalendarAttendancePageState createState() => _CalendarAttendancePageState();
+}
+
+class _CalendarAttendancePageState extends State<CalendarAttendancePage> {
+  bool _animate = false;
+  bool _isAttendanceSelected = true;
+  bool _isBackNavigation = false;
```

Comment:
create object / class that contains animation related variables, for better readability